test/test_ops.py

```diff
@@ -741,6 +741,18 @@ def test_backward(self, device, dtype, op):
             kwargs = sample.kwargs
             composite_compliance.check_backward_formula(op, args, kwargs)
 
+    @unittest.skipIf(IS_FBCODE or IS_SANDCASTLE, '__torch_dispatch__ does not work in fbcode')
+    @ops([op for op in op_db if op.supports_autograd], allowed_dtypes=(torch.float,))
+    def test_forward_ad(self, device, dtype, op):
+        if not op.supports_forward_ad:
```

Comment:
Is there a way to verify that forward ad is not supported instead of just skipping? I recently fixed a lot of skips that were actually working because they weren't verified

Reply:
cc @soulitzer -- do we automatically verify `op.supports_forward_ad` in the forward-mode AD tests?

Reply:
forward_ad test verify if `supports_forward_ad` is set correctly or not.

https://github.com/pytorch/pytorch/blob/1324410f2ec3a1ed0bfc3f0b689c861167170497/test/test_ops_gradients.py#L236-L243

Reply:
Great. nvm then